src/Button/Button.stories.tsx

```diff
@@ -0,0 +1,326 @@
+import React from 'react'
+import {View, Text, StyleSheet, ScrollView} from 'react-native'
+import type {ComponentMeta, ComponentStory} from '@storybook/react'
+
+import {Button} from './Button'
+import {metrics} from '../helpers/metrics'
+
+export default {
+  title: 'components/MyButton',
+  component: Button,
+  argTypes: {
+    backgroundColor: {control: 'color'},
+  },
+} as ComponentMeta<typeof Button>
+
+export const ButtonComponent: ComponentStory<typeof Button> = () => {
+  const imgUrl = 'https://kenh14cdn.com/203336854389633024/2023/1/17/photo-12-1673980290121902612775.jpeg'
```

Comment:
Should not use external link, it may be erased and out example would be corrupted

Reply:
same the above issue